setup/generic/install.sh

```diff
@@ -127,7 +127,7 @@ fi
 # Copy the sample config file from github
 if [ ! -e /boot/teslausb_setup_variables.conf ] && [ ! -e /root/teslausb_setup_variables.conf ]
 then
-  while ! curl -o /boot/teslausb_setup_variables.conf https://raw.githubusercontent.com/marcone/teslausb/main-dev/pi-gen-sources/00-teslausb-tweaks/files/teslausb_setup_variables.conf.sample
+  while ! curl -o /boot/teslausb_setup_variables.conf https://raw.githubusercontent.com/MagoKimbra/teslausb/main-dev/pi-gen-sources/00-teslausb-tweaks/files/teslausb_setup_variables.conf.sample
```

Comment:
Don't change the repo

Reply:
As before, while I am testing it must download the files I modified.